examples/templates/Example.js

```diff
@@ -206,6 +217,32 @@
             if ( optionalArgs )
                 osg.extend( optionsDebug, optionalArgs );
 
+            if ( !this._programRTT ) {
```

Comment:
shadow wasn't using example, now it does, but shadow has its own version of debug RTT which was using the shader program.